modules/monitoring-limiting-scrape-samples-in-user-defined-projects.adoc

```diff
@@ -1,15 +1,23 @@
 // Module included in the following assemblies:
 //
-// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+// * monitoring/configuring-the-monitoring-stack.adoc
 
-[id="limiting-scrape-samples-in-user-defined-projects_{context}"]
-= Limiting scrape samples in user-defined projects
+//[id="limiting-scrape-samples-in-user-defined-projects_{context}"]
+[id="controlling-the-impact-of-unbound-attributes-in-user-defined-projects_{context}"]
+= Controlling the impact of unbound metrics attributes in user-defined projects
 
-Labels are used to define attributes for metrics in the form of key-value pairs. A unique time series is created for every key-value pair. Adding many values to labels can result in an exponential increase in the number of time series created. This can impact Prometheus performance and can consume a lot of disk space.
+Developers can create labels to define attributes for metrics in the form of key-value pairs. The number of potential key-value pairs corresponds to the number of possible values for an attribute. An attribute that has an unlimited number of potential values is called an unbound attribute. For example, a `customer_id` attribute is unbound because it has an infinite number of possible values.
 
-You can prevent this by limiting the number of samples that can be accepted per target scrape in user-defined projects. You should also create alerts that fire when the target cannot be scraped or when a scrape sample threshold is reached.
+.The impact of using many unbound attributes in labels
```

Comment:
Same here, even though this isn't a procedure module. I would add this as a note or some other method. Using that heading above paragraphs makes it look like everything below falls under it, not just a single paragraph.

Reply:
I have removed the dot heading in this file also, as per my other response.